packages/maskbook/src/extension/browser-action/UI.tsx

```diff
@@ -132,20 +132,22 @@ function BrowserActionUI() {
                             sns: ui.networkIdentifier,
                         })}
                     </Typography>
-                    <Box
-                        sx={{
-                            display: 'flex',
-                        }}>
-                        <Button
-                            className={classes.button}
-                            variant="text"
-                            onClick={() => {
-                                if (Flags.no_web_extension_dynamic_permission_request) return
-                                requestSNSAdaptorPermission(ui).then(checkPermission)
+                    {ui.networkIdentifier === 'localhost' || identities.length === 0 ? null : (
+                        <Box
+                            sx={{
+                                display: 'flex',
                             }}>
```

Comment:
```suggestion
                            sx={{ display: 'flex' }}>
```

Reply:
If our linters/formatters don't complain about it, we can just let it go.

Indeed, if these lines get joined, the code can be accepted by prettier, but prettier doesn't provide any option to join the lines.

I like the single-line object too. It's clear and clean. If only we can find a solution to handle this automatically. I come up with an idea: we can minify the code (without mangling variable names or removing comments), then format the code with prettier, and then we can get the object joined.